### PR TITLE
Fix websub configs

### PR DESCRIPTION
**Pull request by BilalRifas**
## Purpose
Fix websub configs

### CODE REVIEW

File: examples/websub-internal-hub-sample/publisher.bal [hunk 1]
```diff
@@ -26,13 +28,14 @@ public function main() {
...
+    if (publishResponse is error) {
+        log:printError("Error notifying hub:", err = publishResponse);
+    } else {
+        io:println("Update notification successful!");
     }
 
     // Make sure the service is running until the subscriber receives the update notification.
     runtime:sleep(2000);
-}
+}
```
Comment (pamod) [inline examples/websub-internal-hub-sample/publisher.bal]: let's add a new line here

File: examples/websub-internal-hub-sample/subscriber.bal [hunk 2]
```diff
@@ -17,34 +17,36 @@ endpoint websub:Listener websubEP {
...
-            log:printInfo("Intent verified for subscription request");
+            io:println("Intent verified for subscription request");
         } else {
-            log:printWarn("Intent verification denied for subscription request");
+            io:println("Intent verification denied for subscription request");
         }
         var result = caller->respond(untaint response);
 
-        if (result is error) {
+        if (result is error){
```
Comment (Bhashinee) [inline examples/websub-internal-hub-sample/subscriber.bal]:
```suggestion
        if (result is error) {
```

File: examples/websub-remote-hub-sample/subscriber.bal [hunk 3]
```diff
@@ -2,11 +2,10 @@
 import ballerina/http;
 import ballerina/log;
 import ballerina/websub;
+import ballerina/io;
 
 // The endpoint to which the subscriber service is bound.
-endpoint websub:Listener websubEP {
-    port: 8181
-};
+listener websub:Listener websubEP = new (8181);
```
Comment (Bhashinee) [inline examples/websub-remote-hub-sample/subscriber.bal]:
```suggestion
listener websub:Listener websubEP = new(8181);
```

File: examples/websub-hub-client-sample/publisher.bal [hunk 4]
```diff
@@ -3,22 +3,23 @@
 import ballerina/io;
 import ballerina/runtime;
 import ballerina/websub;
+import ballerina/log;
 
-public function main() {
+public function main(string... args) {
```
Comment (MaryamZi) [inline examples/websub-hub-client-sample/publisher.bal]: Why are we re-adding `args`? We are not using them right? Check `examples/websub-remote-hub-sample/publisher.bal` too.

File: examples/websub-hub-client-sample/publisher.bal [hunk 5]
```diff
@@ -3,22 +3,23 @@
...
 
     // Register a topic at the hub.
     var registrationResponse = webSubHub.registerTopic(
                                             "http://websubpubtopic.com");
-    match (registrationResponse) {
-        error webSubError => io:println("Error occurred registering topic: "
-                                        + webSubError.message);
-        () => io:println("Topic registration successful!");
+    if (registrationResponse is error) {
+        log:printError("Error occurred registering topic:", err = registrationResponse);
```
Comment (MaryamZi) [inline examples/websub-hub-client-sample/publisher.bal]: We usually use `io:println` with `main` files and `log` with services. Shall we stick to the original?

File: examples/websub-hub-client-sample/subscription_change_client.bal [hunk 6]
```diff
@@ -19,15 +16,13 @@ public function main() {
...
-        websub:SubscriptionChangeResponse subscriptionChangeResponse => {
-            io:println("Subscription Request successful at Hub ["
-                    + subscriptionChangeResponse.hub + "] for Topic ["
-                    + subscriptionChangeResponse.topic + "]");
-        }
-        error e => {
-            io:println("Error occurred with Subscription Request: ", e);
-        }
+    if (response is websub:SubscriptionChangeResponse) {
+            io:println("Subscription Request successful at Hub [" + response.hub + "] for Topic [" +
```
Comment (MaryamZi) [inline examples/websub-hub-client-sample/subscription_change_client.bal]: double spaces?

File: examples/websub-hub-client-sample/publisher.bal (hunk 5 above)
Comment (MaryamZi) [reply examples/websub-hub-client-sample/publisher.bal]: Please check in all places.

File: examples/websub-remote-hub-sample/subscriber.bal [hunk 7]
```diff
@@ -17,33 +16,34 @@ endpoint websub:Listener websubEP {
...
+    resource function onIntentVerification(websub:Caller caller,
+                                           websub:IntentVerificationRequest request) {
+        // Build the response to the intent verification request that was received for subscription.
+        http:Response response = request.buildSubscriptionVerificationResponse("http://websubpubtopic.com");
         if (response.statusCode == 202) {
-            log:printInfo("Intent verified for subscription request");
+            io:println("Intent verified for subscription request");
         } else {
-            log:printWarn("Intent verification denied for subscription request");
+            io:println("Intent verification denied for subscription request");
```
Comment (Bhashinee) [inline examples/websub-remote-hub-sample/subscriber.bal]: Is there any reason for changing log:printWarn to io:println?

**Pull request merged**